psalm-baseline.xml

```diff
@@ -1593,6 +1593,9 @@
       <code>$container</code>
       <code>$container</code>
     </MixedAssignment>
+    <MixedInferredReturnType occurrences="1">
+      <code>($name is class-string ? T : HelperInterface|callable)</code>
+    </MixedInferredReturnType>
```

Comment:
Wait, psalm can't work with this? :thinking:

Reply:
Ah, it cannot infer the return type, but the declared one is still valid for consumers, right?

Reply:
Yes!